manager/gui/src/components/dashboard/LrDashboardPerformanceCard.vue

```diff
@@ -70,12 +97,22 @@ export default class LrDashboardPerformanceCard extends Vue {
 }
 </script>
 <style lang="scss" scoped>
-.performance-cards-container {
-  display: flex;
-  flex-direction: column;
+.info-card-container {
+  border: 1px solid #e5e5e5;
+  padding: 20px 20px;
```

Comment:
I suggest using 'em' for padding and margin, 'rem' for font size and 'px' for border as it is the good practice. We should also create a task to refactor the existing code.

FYI: https://dev.to/datmt/use-em-for-padding-and-margin-54bi#:~:text=The%20reason%20(to%20use%20em,size%20like%20while%20using%20em.

Reply:
Changes done